packages/date-picker/src/DatePicker/components/Calendar/Calendar.scss

```diff
@@ -0,0 +1,91 @@
+@import "~@kaizen/design-tokens/sass/typography";
+@import "~@kaizen/design-tokens/sass/color";
+@import "~@kaizen/design-tokens/sass/spacing";
+@import "~@kaizen/design-tokens/sass/border";
+@import "~@kaizen/design-tokens/sass/shadow";
+
+// Vars
+$border-radius: 7px;
+
+// Calendar styles
+.calendar {
+  background-color: $color-white;
+  font-family: $typography-heading-4-font-family;
+  z-index: 1;
+
+  .wrapper {
+    padding-bottom: 0;
+  }
+
+  .month {
+    margin: $spacing-md;
+    border-collapse: separate;
+    border-spacing: 1px;
+  }
+
+  .months {
+    box-shadow: $shadow-large-box-shadow;
+    border-radius: $border-solid-border-radius;
+  }
+
+  .caption {
+    text-align: center;
+    font-size: $typography-heading-4-font-size;
+    font-weight: $typography-heading-4-font-weight;
+    line-height: $typography-heading-4-line-height;
+    letter-spacing: $typography-heading-4-letter-spacing;
+    color: $color-purple-800;
+    margin: 0 0 $spacing-xs;
+  }
+
+  .weekday {
+    font-size: $typography-paragraph-body-font-size;
+    font-weight: $typography-paragraph-body-font-weight;
+    line-height: $typography-paragraph-body-line-height;
```

Comment:
Usually when we use these typography tokens, we use all of them (with the odd case where the `line-height` doesn't work well). In order words, this one is missing `letter-spacing`, `font-family` and possibly `line-height` (and there's other similar cases in this file).

It may not seem important right now because everything that's not `display-0` has Inter as the font-family, but if we ever decide to change fonts for headings but leave Inter for paragraph, this will end up with the heading font family.